Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Registration.hs

```diff
@@ -197,7 +197,8 @@ createDriverDetails personId merchantId = do
             createdAt = now,
             updatedAt = now,
             compAadhaarImagePath = Nothing,
-            availableUpiApps = Nothing
+            availableUpiApps = Nothing,
+            optForRental = False
```

Comment:
By default, it should be true or false should be merchant wise config like downgrade